src/js/tracking-wallet.js

```diff
@@ -747,6 +747,10 @@
     }
   };
 
+  var getUserId = function () {
+    return window.mixpanel.get_distinct_id();
```

Comment:
What do you think about adding the check to ensure that mixpanel is loaded to prevent TypeError error.
```suggestion
if (window.mixpanel && window.mixpanel.get_distinct_id())
    return window.mixpanel.get_distinct_id();
```

Reply:
Sure, added!